src/runner/nextflow.config

```diff
@@ -0,0 +1,6 @@
+manifest {
+  nextflowVersion = '!>=20.12.1-edge'
```

Comment:
I think we might need a more recent verison.. but I have no clue which one...

Reply:
I suggest to leave this for now.

Reply:
Sure.